docs/sdk/src/guides/enable_elastic_scaling.rs

```diff
@@ -0,0 +1,235 @@
+#![doc = docify::embed!("../../cumulus/polkadot-omni-node/lib/src/nodes/aura.rs", slot_based_colator_import)]
+//!
+//! 2. Modify the `ParachainBlockImport` and `Service` type definitions:
+//! ```ignore
+//! type ParachainBlockImport = TParachainBlockImport<
+//! 	    Block,
+//! 	    SlotBasedBlockImport<Block, Arc<ParachainClient>, ParachainClient>,
+//! 	    ParachainBackend,
+//! >;
+//! ```
+//!
+//! ```ignore
+//! pub type Service = PartialComponents<
+//!     ParachainClient,
+//!     ParachainBackend,
+//!     (),
+//!     sc_consensus::DefaultImportQueue<Block>,
+//!     sc_transaction_pool::TransactionPoolHandle<Block, ParachainClient>,
+//!     (
+//!         ParachainBlockImport,
+//!         SlotBasedBlockImportHandle<Block>,
+//!         Option<Telemetry>,
+//!         Option<TelemetryWorkerHandle>,
+//!     ),
+//! >;
+//! ```
+//!
+//! 3. In `new_partial()`:
+//!     - Instantiate the `SlotBasedBlockImport` and pass the returned `block_import` value to
+//!       `ParachainBlockImport::new` and the returned `slot_based_handle` to the `other` field of
+//!       the `PartialComponents` struct.
+//!      
+//!      ```ignore
+//!      let (block_import, slot_based_handle) = SlotBasedBlockImport::new(
+//!          client.clone(),
+//!          client.clone()
+//!      );
+//!      let block_import = ParachainBlockImport::new(block_import.clone(), backend.clone());
+//!      ```
+//!
+//!      ```ignore
+//!      Ok(PartialComponents {
+//! 		backend,
+//! 		client,
+//! 		import_queue,
+//! 		keystore_container,
+//! 		task_manager,
+//! 		transaction_pool,
+//! 		select_chain: (),
+//! 		other: (block_import, slot_based_handle, telemetry, telemetry_worker_handle),
+//! 	 })
+//!      ```
+//!
+//! 2. In `start_consensus()`:
+//!     - Remove the `overseer_handle` and `relay_chain_slot_duration` params (also remove the
+//!     `OverseerHandle` type import if it’s not used elsewhere).
+//!     - Add a new parameter for the block import handle: `block_import_handle:
+//!       SlotBasedBlockImportHandle<Block>`
+//!     - Rename `AuraParams` to `SlotBasedParams`, remove the `overseer_handle` and
+//!     `relay_chain_slot_duration` fields and add a `slot_drift` field with a value of
+//!     `Duration::from_secs(1)`. Also add a `spawner` field initialized to
+//!     `task_manager.spawn_handle()` and pass in the `block_import_handle` param.
+//!     - Replace the `aura::run` with the `slot_based::run` call and remove the explicit task
+//!       spawn:
+#![doc = docify::embed!("../../cumulus/polkadot-omni-node/lib/src/nodes/aura.rs", launch_slot_based_collator)]
+//!
+//! 3. In `start_parachain_node()`, destructure `slot_based_handle` from `params.other`. Remove the
+//!    `overseer_handle` and `relay_chain_slot_duration` params passed to `start_consensus` and pass
+//!    in the `slot_based_handle`.
+//!
+//! ### Phase 2 - Configure core selection policy in the parachain runtime
```

Comment:
Phase 2 assumes candidate receipt v2 feature bit was enabled.
This phase will change after the feature bit is enabled on all networks and a form of https://github.com/paritytech/polkadot-sdk/pull/6939 is merged